src/components/ux-table-body/ux-table-body.hbs

```diff
@@ -0,0 +1,3 @@
+<tbody {{#if class}}class="{{.class}}"{{/if}} {{#if id}}id="{{.id}}"{{/if}}>
```

Comment:
Either you should always use "class" or you should use ".class" (same goes for all other variables) in the above line and lots of other lines in other files you mix and match the styles.

Either the line should be:

```javascript
<tbody {{#if class}}class="{{class}}"{{/if}} {{#if id}}id="{{id}}"{{/if}}>
```

or

```javascript
<tbody {{#if .class}}class="{{.class}}"{{/if}} {{#if .id}}id="{{.id}}"{{/if}}>
```

My preference is for the first form (less verbose) but I'll check if there is a reason for the other form.